package.json

```diff
@@ -10,8 +10,8 @@
     "test": "npm run test-unit && npm run lint",
     "test-unit": "mocha --require intelli-espower-loader test/unit/*.test.js",
     "test-all": "mocha --require intelli-espower-loader test/**/*.test.js",
-    "coverage": "nyc npm test",
-    "coverage-all": "nyc npm run test-all",
+    "coverage": "nyc --reporter=lcov --reporter=text --reporter=text-summary npm test",
```

Comment:
Maybe add `--reporter=html`?

Reply:
Lcov generates both `html` and `lcov`. I assume that `lcov` is only useful for you to `codecov`.

I change it, no problem.